cdap-standalone/src/main/java/co/cask/cdap/StandaloneMain.java

```diff
@@ -386,6 +402,7 @@ public static void main(String[] args) {
         System.err.println("Failed to start Standalone CDAP");
         e.printStackTrace(System.err);
       }
+      main.stopExternalProcesses();
```

Comment:
It might have been because we had only a single process, and since this is in a catch block, rather than attempting a clean shutdown (which might or might not work), just kill the entire process.